src/multi_vector_simulator/utils/data_parser.py

```diff
@@ -373,6 +383,13 @@ def convert_epa_params_to_mvs(epa_dict):
                             "this warning in data_parser.py and remove the warning and the 7 "
                             "lines of code above it as well"
                         )
+
+                if asset_group == ENERGY_CONSUMPTION:
+                    if DSM not in dict_asset[asset_label]:
```

Comment:
`TYPE_ASSET` does not need the default value? For as long as we have not removed them properly? (#376)